nncf/common/pruning/operations.py

```diff
@@ -223,6 +227,69 @@ def mask_propagation(cls, node: NNCFNode, graph: NNCFGraph,
         node.data['output_mask'] = result_mask
 
 
+class SplitPruningOp(BasePruningOp):
+    @classmethod
+    def match_multiple_output_masks(cls, output_masks: List[SymbolicMask], output_edges: List[NNCFGraphEdge],
+                                    chunk_axis: int) -> Dict['str', SymbolicMask]:
+        """
+        Match multiple input mask to each next nodes.
+
+        :param output_masks: Given output masks.
+        :param output_edges: Given output edges of the node.
+        :param chunk_axis: Given the axis on which operation was performed.
+        :return: Matched output mask for each next node.
+        """
+        next_nodes = [edge.to_node for edge in output_edges]
+        result_masks = {node.node_name: None for node in next_nodes}
```

Comment:
```suggestion
```

Reply:
Fix. I could not think this part. Thanks.